setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='gridmet',
-    version="0.0.3.6",
+    version="0.0.3.8",
     url='https://gitlab-int.rc.fas.harvard.edu/rse/francesca_dominici/tools/gridmet',
```

Comment:
Good catch, it is certainly obsolete.

Reply:
As to gridmet an pollution, the pipelines are different in that they are retrieving raw data from different sources. In case of gridMET it is Google Earth an for exposure - local files or s3 buckets.